### DISCUSSION TITLE
can you use the VLOOKUP formula?

### DISCUSSION THREAD

**Discussion by fanovoa** [exceljs/exceljs#2139] [Q&A]
Hi,
is it possible to implement the vlookup formula?  I read the manual but I did not find the reference

**Comment by Devel-JD** [comment]
We can use VLOOKUP in the formula by using `getCell('cellName').value = { formula: VLOOKUP(...) }`. One thing to notice is that we use semicolon `;` to separate the parameter in Excel, but we use comma `,` instead in exceljs formula. Ex. `{ formula: VLOOKUP(A12,'OtherSheet'!$B$2:$D$1048576,3,0)}`
